azure-pipelines.yml

```diff
@@ -199,15 +213,11 @@ stages:
                 $signAssemblies = [System.Convert]::ToBoolean("$(IS_RELEASE_BRANCH)") # the variable is a string, we need a boolean
                 Package-NetFrameworkScanner -SignAssemblies $signAssemblies
                 Package-NetScanner -SignAssemblies $signAssemblies
-              pwsh: true
```

Comment:
What did this do? Why is it not needed anymore?

Reply:
It runs the script on Powershell Core instead of regular (Windows-only) Powershell.
@sebastien-marichal removed it. My guesses for the reasons:
1. Maybe it didn't run on the new image because Powershell Core isn't installed on the image.
2. Or just to keep it consistent: none of the other Powershell tasks use this parameter.